modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -19,19 +19,27 @@ Cluster Logging Operator and Elasticsearch Operator.
 
 .Prerequisites
 
-Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
-requires its own storage volume.
+You are provided with a maximum of 600GiB of persistent storage exclusively
```

Comment:
The prereqs should be in an ordered list.

maybe

```
* Create storage volumes for each Elasticsearch node. Your {product-title} cluster includes 600 GiB of persistent storage that is exclusively for deploying Elasticsearch for cluster logging.
```

Do these exist by default? Does a user need to create them before they start? Do they know how many nodes to create up front?

Reply:
Honestly, I think we should just do away with the `.Prerequisites`. Nothing in there is a prereq, now that I'm looking at it. I think it could be a continuation of the statement above the note. This is just informational text.  What do you think?

Reply:
It depends on if the user needs to actually do anything to prepare these items or if it's mostly FYI. If it's FYI, I agree that removing the prereqs section would be better.

Reply:
Just FYI